scss/_functions.scss

```diff
@@ -271,3 +271,9 @@ $_luminance-list: .0008 .001 .0011 .0013 .0015 .0017 .002 .0022 .0025 .0027 .003
   }
   @return $result;
 }
+
+@mixin validate-utilities-field($utility, $utility-name, $field) {
```

Comment:
Private mixin name should start with a `_`.

I don't see the point of this mixin however, why not simply throw the warning in `generate-utility`?

Reply:
A separate mixin is needed to maintain consistency and avoid code duplication. But if the new code outside the generation file is not needed, then I will transfer it